### ISSUE TITLE
Update endpoint used by github action

### ISSUE THREAD

**Issue by dequejenn** [dequelabs/axe-devhub-action#134]
Need to update the endpoint used byt he github action to get the a11y threshold overage to use https://github.com/dequelabs/walnut/issues/7236. In addition, the github action should be updated to show those new fields:

In the PR comment:
1. Add in `axe DevHub found X accessibility violations over your a11y threshold.` where `X` is the number returned byt he enw endpoint
2. Update the link to point to the URL returned by the new endpoint
![image](https://github.com/dequelabs/axe-devhub-action/assets/112327890/b0be1ab8-a971-4925-a533-c39bb6b57cd1)

In the Action display:
3. Update the number of violations over a11y threshold with the number returned from the new endpoint (line 33 in example screenshot below)
4. Update URL with URL returned by the new endpoint (like 34 in the example screenshot below)
![image](https://github.com/dequelabs/axe-devhub-action/assets/112327890/5c221727-7804-40c2-9658-b0205e7b4052)
